scripts/constants.py

```diff
@@ -0,0 +1,27 @@
+import os
+from enum import Enum
+from pathlib import Path
+
+SAMPLES_REPO = 'Mojang/bedrock-samples'
+
+LINE = '-' * 20
+
+ROOT = Path('../')
+TAGS_PATH = Path('../tags.json')
+
+CACHE_PATH = Path('./cache')
+CACHE_PATH.mkdir(exist_ok=True, parents=True)
+
+TMP_PATH = Path('./tmp')
+TMP_PATH.mkdir(exist_ok=True, parents=True)
+
+IS_ACTIONS = 'GITHUB_ACTIONS' in os.environ
+
+class Version(Enum):
```

Comment:
I'm not a huge fan of this 'Enum' + dictionary. But I need to read further to see how it's used.